driver-sync/src/examples/documentation/DocumentationSamples.java

```diff
@@ -41,11 +45,19 @@
 import static com.mongodb.ClusterFixture.serverVersionAtLeast;
 import static com.mongodb.client.Fixture.getDefaultDatabaseName;
 import static com.mongodb.client.Fixture.getMongoClient;
+import static com.mongodb.client.model.Accumulators.sum;
```

Comment:
Not sure what to do about these imports.  In a previous commit I must have optimized imports and all the `imports required for change streams` comments were moved to the bottom.

But now many of the imports are used by more than one example, and we can't really repeat them.